src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx

```diff
@@ -680,6 +680,40 @@ Set these options in the `common` stanza. To [override](#System_Properties) any
     ```
   </Collapser>
 
+  <Collapser
+    id="legacy_async_api_skip_suspend"
+    title="legacy_async_api_skip_suspend"
+  >
+    <table>
+      <tbody>
+      <tr>
+        <th>
+          Type
+        </th>
+
+        <td>
+          Boolean
+        </td>
+      </tr>
+
+      <tr>
+        <th>
+          Default
+        </th>
+
+        <td>
+          False
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    In rare cases, a memory leak could occur with instrumentation that uses the legacy async API to track the lifecycle
+    of async servlets or Jetty Continuations. The memory leak would manifest as leaked transactions that were suspended
+    but never resumed. A configurable workaround has been introduced to prevent such a scenario from occurring. It is
```

Comment:
@deleonenriqueta I don't think it's a deterrent from pushing this live but what is the workaround? We should link it somewhere.

Reply:
@homelessbirds @deleonenriqueta Maybe reword "A configurable workaround has been introduced to prevent such a scenario from occurring." to instead be "This configuration acts as a workaround to prevent such a scenario from occurring."

Reply:
Updated the descriptor with the suggested rewording. @homelessbirds @obenkenobi